config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  resources :recipes
   resources :apartments
```

Comment:
I am seeing resources for apartments in here. Is there supposed to be apartments?